### DISCUSSION TITLE
What is the legal status of the project?

### DISCUSSION THREAD

**Discussion by CognitiveCodes** [daveshap/ACE_Framework#26] [General]
Are there some investors and/or plans as for hiring people? I'm working on my project as a weird kind of hobby and I don't have anything against that but I had just a talk with professional dev who wanted to work on the project for 2000$/month. Obviously I can't afford that but I guess the ACE is popular enough to get a sponsor and start hiring people legally...?

**Comment by daveshap** [comment]
We're in talks with several potential sponsors, but it will be more of a corporate project rather than a grant. The goal is to establish reference architectures and best practices
